rules/windows/defense_evasion_code_signing_policy_modification_builtin_tools.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2023/01/31"
+integration = ["endpoint", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/01/31"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to disable/modify the code signing policy through the registry. Code signing provides 
+authenticity on a program, and grants the user with the ability to check whether the program has been tampered with. 
+By allowing the execution of unsigned or self-signed code, threat actors can craft and execute malicious code. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Code Signing Policy Modification Through Built-in tools"
+risk_score = 21
+rule_id = "b43570de-a908-4f7f-8bdb-b2df6ffd8c80"
+severity = "low"
+tags = [
+    "Elastic",
+    "Host",
+    "Windows",
+    "macOS",
+    "Threat Detection",
+    "Defense Evasion",
+    "Elastic Endgame",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and 
+(
+  /* Windows */
+  (process.name: "bcdedit.exe" or process.pe.original_file_name == "bcdedit.exe") and
+  ((process.args: "-set" and process.args: "TESTSIGNING" or process.args: "nointegritychecks") or
+  (process.args: "-set" and process.args: "loadoptions" or process.args: "DISABLE_INTEGRITY_CHECKS")) or
```

Comment:
For some reason that didn't return the behavior I was expecting, because I was missing events. Reason ended up being that I used /set and -set commands while testing, instead of only -set. Your query would not pick that up, while mine would. But now I fixed it, to first look for /set and set, and then use your suggestion to make it cleaner. Thanks!